pkg/controller/cloneset/cloneset_controller.go

```diff
@@ -213,7 +212,7 @@ func (r *ReconcileCloneSet) doReconcile(request reconcile.Request) (res reconcil
 	}
 
 	// If scaling expectations have not satisfied yet, just skip this reconcile.
-	if scaleSatisfied, unsatisfiedDuration, scaleDirtyPods := clonesetutils.ScaleExpectations.SatisfiedExpectations(request.String()); !scaleSatisfied {
+	if scaleSatisfied, unsatisfiedDuration, scaleDirtyPods := clonesetutils.ScaleExpectations.SatisfiedExpectations(request.Namespace); !scaleSatisfied {
```

Comment:
It will cause all CloneSets in a namespace paused, if one Pod in the namespace has not satisfied.
Any other good ideas?